src/tools/debugger/nodes/BreakpointNode.java

```diff
@@ -0,0 +1,45 @@
+package tools.debugger.nodes;
+
+import com.oracle.truffle.api.Assumption;
+import com.oracle.truffle.api.dsl.Cached;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.source.SourceSection;
+
+import som.VM;
+import tools.SourceCoordinate;
+import tools.debugger.session.BreakpointEnabling;
+import tools.debugger.session.MessageReceiveBreakpoint;
+
+/**
+ *
+ * Node to represent a breakpoint, which contains two possible states, enable or disable.
```

Comment:
Please change to: `Node to represent a breakpoint at the AST level. It has to two possible states: enabled or disabled.`